AutomatedTesting/Gem/PythonTests/editor/EditorScripts/Sample_Editor_Tests.py

```diff
@@ -0,0 +1,143 @@
+    from editor_python_test_tools.utils import TestHelper as helper
+    import editor_python_test_tools.hydra_editor_utils as hydra
+
+    import azlmbr.math as math
+    import azlmbr.asset as asset
+    import azlmbr.bus as bus
+    import azlmbr.editor as editor
+    import azlmbr.entity as entity
+    import azlmbr.legacy.general as general
+    import azlmbr.object
+
+    def search_entity(entity_to_search, entity_name):
+        """
+
+        :param entity_to_search: entity to be searched
+        :param entity_name: name of the entity used in the set command
+        :return: True if entity id exists in the entity_list
+                 False if entity id does not exist in the entity_list
+        """
+        entity_list = []
+        entity_search_filter = entity.SearchFilter()
+        entity_search_filter.names = entity_name
+        entity_list = entity.SearchBus(bus.Broadcast, 'SearchEntities', entity_search_filter)
+        if entity_list:
+            if entity_to_search in entity_list:
+                return True
+            return False
+        return False
+
+    # 01. create_level
+
+    test_level = 'Simple'
+    general.open_level_no_prompt(test_level)
+    Report.result(Tests.create_level, general.get_current_level_name() == test_level)
+
+    # 02. load existing level - skipping this since this could alter existing level that other test depends on
+
+    # 03. create_entity and set name
+    # Delete any exiting entity and Create a new Entity at the root level
+    search_filter = azlmbr.entity.SearchFilter()
+    all_entities = entity.SearchBus(azlmbr.bus.Broadcast, "SearchEntities", search_filter)
+    editor.ToolsApplicationRequestBus(bus.Broadcast, "DeleteEntities", all_entities)
+    parent_entity = editor.ToolsApplicationRequestBus(bus.Broadcast, "CreateNewEntity", entity.EntityId())
+    Report.result(Tests.create_entity, parent_entity.IsValid())
+
+    # Setting a new name
+    parent_entity_name = "Parent_1"
+    editor.EditorEntityAPIBus(bus.Event, 'SetName', parent_entity, parent_entity_name)
+    Report.result(Tests.set_entity_name,
+                  editor.EditorEntityInfoRequestBus(bus.Event, 'GetName', parent_entity) == parent_entity_name)
+
+    # 04. Creating child Entity and setting name to above created parent entity
+    child_1_entity = editor.ToolsApplicationRequestBus(bus.Broadcast, 'CreateNewEntity', parent_entity)
+    Report.result(Tests.create_child_entity, child_1_entity.IsValid())
+    child_entity_name = "Child_1"
+    editor.EditorEntityAPIBus(bus.Event, 'SetName', child_1_entity, child_entity_name)
+    Report.result(Tests.set_entity_name,
+                  editor.EditorEntityInfoRequestBus(bus.Event, 'GetName', child_1_entity) == child_entity_name)
+
+    # 05. delete_Child_entity
+    editor.ToolsApplicationRequestBus(bus.Broadcast, 'DeleteEntityById', child_1_entity)
+    Report.result(Tests.delete_entity, search_entity(child_1_entity, "Child_1") == False)
```

Comment:
with editor_entity_utils you can
child_1_entity.delete()
not child_1_enity.exists()

Reply:
fixed by using hydra_editor_utils.py def find_entity_by_name
changes are in latest commit